massa-graph/src/export_active_block.rs

```diff
@@ -32,67 +32,6 @@ pub struct ExportActiveBlock {
     pub production_events: Vec<(u64, Address, bool)>,
 }
 
-impl From<&ActiveBlock> for ExportActiveBlock {
-    fn from(a_block: &ActiveBlock) -> Self {
-        ExportActiveBlock {
-            block: a_block.block.clone(),
-            parents: a_block.parents.clone(),
-            children: a_block.children.clone(),
-            dependencies: a_block.dependencies.clone(),
-            is_final: a_block.is_final,
-            block_ledger_changes: a_block.block_ledger_changes.clone(),
-            roll_updates: a_block.roll_updates.clone(),
-            production_events: a_block.production_events.clone(),
-        }
-    }
-}
-
-impl TryFrom<ExportActiveBlock> for ActiveBlock {
```

Comment:
Note: removed those because shared storage is not easily available from here, and also because there weren't used in more than one place anyway, so this is now simply code in `block_graph`, where shared storage is available.

Reply:
It's needed in the tests. Is that possible to have it maybe with a cfg tests ? Also I don't understand why you need shared storage to make a conversion ?

Reply:
I have made the modification on my PR for tests because I needed it you can check and merge it here even if it's not finish so that you have the blockId instead of block here. PR : https://github.com/massalabs/massa/pull/2383 @gterzian

Reply:
great, thanks